content/docs/14.best-practices/7.outputs.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Managing and purging flow outputs
+title: Managing and Purging Flow Outputs
 icon: /docs/icons/best-practices.svg
 ---
 
-Best practices for handling flow outputs incl. purging large outputs and conditionally returning outputs.
+Best practices for handling flow outputs, including purging large outputs and conditionally returning outputs.
 
 ## Conditionally returning outputs
 
-When you have a flow that can return different outputs based on some condition, you can use a Pebble expression in the `outputs` section to conditionally return the output of task `A` if it wasn't skipped, and the output of task `B` otherwise.
+When a flow can return different outputs depending on certain conditions, you can use a Pebble expression in the `outputs` section. This allows you to conditionally return the output of task `A` if it wasn’t skipped, or the output of task `B` otherwise.
```

Comment:
I like to avoid using the word ‘Pebble’ and just focus on it being our expressions because of how many unique bits there are, but curious what you think.

```suggestion
When a flow can return different outputs depending on certain conditions, you can use an expression in the `outputs` section. This allows you to conditionally return the output of task `A` if it wasn’t skipped, or the output of task `B` otherwise.
```